homeassistant/components/homepluscontrol/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "domain": "homepluscontrol",
```

Comment:
Our standard is to separate words with underscore in domain names.
```suggestion
  "domain": "home_plus_control",
```

Reply:
OK, noted. Should I apply this change also to the package/directory naming or can those remain without the underscore?

Reply:
The package directory should be named after the domain. :+1:

Reply:
I see that this change is more challenging because of the refactoring and because the repository loses track of the file history. I am planning on changing this in my next commit, now that all of the other review items have been addressed and the checks are all passing.

Reply:
I have completed the refactoring of the component. It is now named: `home_plus_control`.